src/receive_imf/receive_imf_tests.rs

```diff
@@ -2069,7 +2069,7 @@ Chat-Version: 1.0
 
 Message content
 
--- 
+--
```

Comment:
Why remove this whitespace? IIRC, the standard for email footers requires that the whitespace is there. OTOH, Delta Chat seems to recognize the footer just fine.

Reply:
Likely automatically removed by some editor?

It is removed in 3 places, should be reverted before merging.